lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -119,6 +119,28 @@
     - Maximum number of CloudFormation events to fetch from a stack when creating or updating it.
     default: 200
     version_added: "2.7"
+  backoff_delay:
```

Comment:
idle thought: can these be added with a doc fragment instead?

Reply:
Not all module support this. Once they do we can move this to doc fragment.

Reply:
I was thinking about a new doc fragment, like aws_jittered, but it's fine too. I'll approve it for the bot.